### DISCUSSION TITLE
Changing Report Visualization

### DISCUSSION THREAD

**Discussion by crystal-roses** [Data-Centric-AI-Community/fg-data-profiling#1086] [Q&A]
Wondering if anyone has been successful at changing the variable histogram plots to conventional "X vs Y" plots. I would like to use this report to share time dependent data. My input file is an excel sheet that has time in Column 1. I would essentially like a plot of Column 1 vs Column 2, Column 1 vs Column 3, Column 1 vs Column4, etc. instead of a histogram for each column. 

ProfileReport has been great for other datasets, would love to use this in various applications!

Thanks!

**Comment by fabclmnt** [comment]
Hi @crystal-roses this sounds like a feature request :) 

May I ask you to open an issue with this detail? Also, I might have missed the type of visualization you would be expecting. Can you let me know compared to the "Interactions" section of pandas-profiling, what this feature would add and enable in your case?
